bedrock/firefox/views.py

```diff
@@ -542,14 +544,18 @@ def new(request):
                 template = 'firefox/new/batm/scene2.html'
```

Comment:
Should be able to remove `batmprivate` from the list above (line 543) as it's now checked for above.